admin/view.php

```diff
@@ -0,0 +1,152 @@
+<?php
+ob_start();
+session_start();
+
+require 'app/ConfigEnum.php';
+require 'app/DatabaseConfiguration.php';
+require 'app/DatabaseConnection.php';
+require 'app/models/WifiSpots.php';
+require 'app/dao/WifiSpotsDao.php';
+require 'app/handlers/HandleWifiSpots.php';
+
+$isLoggedIn = false;
+$adminUsername = null;
+$v = null;
+$isExist = false;
+if (isset($_SESSION)) {
+    if (isset($_SESSION["authenticated"]) && isset($_SESSION["adminUsername"])) {
+        $isLoggedIn = true;
+        $adminUsername = $_SESSION["adminUsername"];
+
+        if (isset($_GET["itemId"])) {
+            $config = new DatabaseConfiguration(
+                ConfigEnum::DB_HOST,
+                ConfigEnum::DB_PORT,
+                ConfigEnum::DB_NAME,
+                ConfigEnum::DB_USER,
+                ConfigEnum::DB_PASSWORD
+            );
```

Comment:
There instance declarations are rather repetitive, later on for refactoring we might have to find a way to organize these, as the credentials will always be the same.

Reply:
True, you can think of modifying some of these class so that `DatabaseConfiguration` is call once